src/DASCRobots.cpp

```diff
@@ -121,6 +126,7 @@ bool DASCRobot::disarm() {
     VehicleCommand msg;
     msg.timestamp = get_current_timestamp();
     msg.param1 = 0.0;
+    msg.param2 = 21196; // set param2 to 21196 to force arm/disarm operation
```

Comment:
this was needed to force the disarming. not sure why forcing is necessary.

Reply:
what was the error when this param was not added? did you try arming from QGC? Did that work?

Reply:
Arming was fine. 
Disarming wouldn't occur without this forcing.